docs/user_guide/activation_keys/index.md

```diff
@@ -37,11 +37,29 @@ To create a new key,
 - navigate to: Content > Activation Keys
 - click **New Activation Key**
 
-Note the following option:
+![Creating a Host Collection](./activation_key_create.png)
 
-- *Content Host Limit*: This option will control how many Content Hosts may be registered using the key.
+- *Name*: This required option has few restrictions on format. However, since this name is used to identify the activation key to command line tools such as *subscription-manager* including spaces and non-ascii characters will need special consideration in such uses.
```

Comment:
Oh, yea I get what you are trying to say, and given I got stuck on the confusion of that first sentence perhaps relegate it towards the end? or drop it all together? That way users don't get hung up and you could start with the meat of things `This option is used to identify the activation key....`